101-app-configuration/azuredeploy.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "appConfigStoreName": {
+        "type": "string",
+        "metadata": {
+          "description": "App configuration store name."
+        }
+      },
+      "adminPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "Password for the Virtual Machine."
+        }
+      },
+      "location": {
+        "type": "string",
+        "defaultValue": " [resourceGroup().location]",
+        "metadata": {
+          "description": "Location for all resources."
+        }
+      },
+      "vmSize": {
+        "type": "string",
+        "defaultValue": "[listKeyValue(variables('appConfigRef'), variables('appConfigApiVersion'), variables('vmSize')).value]",
```

Comment:
listKeyValue cannot be used here - only in outputs or the properties body of a resource

Reply:
We want to use it in the properties body of a resource, but when I do, the build fails because vmSize must be a parameter. How can we get around that?

Reply:
simplest might be to use a different property for the sample...  IOW, don't use the configstore for the vm size